up42/storage.py

```diff
@@ -0,0 +1,91 @@
+from typing import Dict, List, Union
+
+from tqdm import tqdm
+
+from up42.auth import Auth
+from up42.order import Order
+from up42.asset import Asset
+from up42.utils import get_logger
+from up42.tools import Tools
+
+logger = get_logger(__name__)
+
+
+class Storage(Tools):
```

Comment:
Can remove Tools from Storage, Assets, Orders. The inheritance from Tools is prepared to get deprecated soon for the other classes, so guess don't even need to introduce here.

Reply:
Ah cool, missed this completely, thanks.

Reply:
Removing the Tools inheritance gives out mypy errors in all of the classes, seems like we would need to revisit this some other time.